tests/network/services.go

```diff
@@ -154,64 +149,62 @@ var _ = SIGDescribe("[Serial]Services", func() {
 		})
 
 		Context("with a service matching the vmi exposed", func() {
-			var jobsToCleanup []func() error
+			var err error
```

Comment:
This global is not needed, or is it? You can define it inside `It` right before you use it (to assure that `jobCleanup` is global). That way we make sure that we don't leak error from one test to the other.

Just a  nit though, won't fight for it.

Reply:
Done.